### PR TITLE
Mypy for sqlalchemy-oso

### PR DESCRIPTION
**Pull request by dhatch**

### CODE REVIEW

File: languages/python/oso/setup.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def get_version(rel_path):
     #
     # If using Python 2.6 or earlier, then these have to be included in
     # MANIFEST.in as well.
-    # package_data={"polar": ["policies/*.polar", "policies/*.pol"]},  # Optional
+    package_data={"polar": ["py.typed"], "oso": ["py.typed"]},  # Optional
```
Comment (samscott89) [inline languages/python/oso/setup.py:70]: Oh nice, thanks for working this out.

File: languages/python/sqlalchemy-oso/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 oso~=0.8.2
 sqlalchemy~=1.3.20
+sqlalchemy-stubs
```
Comment (samscott89) [inline languages/python/sqlalchemy-oso/requirements.txt:3]: Is this needed here?

**Review by samscott89 — CHANGES_REQUESTED** [review]
Looks good! Small question about the stubs dependency

File: languages/python/sqlalchemy-oso/requirements.txt (hunk 2 above)
Comment (dhatch) [reply languages/python/sqlalchemy-oso/requirements.txt:3]: Instead of a development dependency?

Comment (dhatch) [reply languages/python/sqlalchemy-oso/requirements.txt:3]: Yeah might work that way.

Comment (dhatch) [reply languages/python/sqlalchemy-oso/requirements.txt:3]: A user would need to install that to use type checking though if this library ships with types.

**Review by samscott89 — APPROVED** [review]
Looks good once its passing CI

File: languages/python/sqlalchemy-oso/setup.cfg [hunk 3]
```diff
@@ -9,6 +9,12 @@ plugins = sqlmypy
 [mypy-pytest.*]
 ignore_missing_imports = True
 
+[mypy-oso.*]
+ignore_missing_imports = True
```
Comment (samscott89) [inline languages/python/sqlalchemy-oso/setup.cfg:13]: Doesn't this defeat the purpose?

**Pull request merged**